lib/host/wasi/wasimodule.cpp

```diff
@@ -65,6 +65,10 @@ WasiModule::WasiModule() : ImportObject("wasi_snapshot_preview1") {
   addHostFunc("sock_recv", std::make_unique<WasiSockRecv>(Env));
   addHostFunc("sock_send", std::make_unique<WasiSockSend>(Env));
   addHostFunc("sock_shutdown", std::make_unique<WasiSockShutdown>(Env));
+  addHostFunc("sock_getsockopt", std::make_unique<WasiSockGetOpt>(Env));
+  addHostFunc("sock_setsockopt", std::make_unique<WasiSockSetOpt>(Env));
+  addHostFunc("sock_get_local", std::make_unique<WasiSockGetLocalAddr>(Env));
```

Comment:
Your function name should be the same as its unix version, e.g. `getpeername`.

Reply:
My function only return address, so I rename it to `getlocaladdr` and `getpeeraddr`.